.github/workflows/release.yaml

```
@@ -17,21 +17,21 @@ on:
 # Configure constants for this workflow.
 env:
   # The base filename of the binary produced by `cargo build`.
-  BINARY: bevy_quickstart
-  # The name to use for the packaged application produced by this workflow.
-  PACKAGE_NAME: bevy_quickstart
-  # The itch.io page to upload to, in the format: `user-name/project-name`.
-  # Comment this out to disable.
-  ITCH_TARGET: the-bevy-flock/bevy-quickstart
-  # The organization or author that owns the rights to the game.
-  OWNER: the-bevy-flock
+  cargo_binary_name: bevy_quickstart
   # The path to the assets directory.
-  ASSETS_DIR: assets
-  # Whether packages produced by this workflow should be uploaded to the Github release.
-  UPLOAD_PACKAGES_TO_GITHUB_RELEASE: true
+  assets: assets
+  # The name to use for the app produced by this workflow.
+  app_name: bevy_quickstart
```

```suggestion
  app_name: Bevy Quickstart
```
This should make it more clear what the difference between this and the other binary name is.

---

Nah you want this to be `bevy_quickstart` so that the package is `bevy_quickstart-windows.zip` with a `bevy_quickstart.exe` binary inside, rather than `Bevy Quickstart-windows.zip` with a `Bevy Quickstart.exe` binary inside.

---

Unless we want to have 2 env variables, one for the MacOS "app name" in the metadata file, and one for the app package / binary filenames.

---

At least on Windows, having executables like "Some Cool Executable.exe" with spaces in-between is not that uncommon.
In any case, it currently is a bit confusing that there are two very similar names at play here. One of them could just be defaulted.
The macOS app name should have an own env var if the others are not supposed to have spaces.

---

It's _allowed_ but very strange to have spaces in a linux executable.